input/drivers/switch_input.c

```diff
@@ -122,50 +192,62 @@ static void switch_input_poll(void *data)
       }
    }
 
-   if (hidMouseButtonsHeld() & MOUSE_LEFT)
-   {
-      sw->mouse_button_left = true;
-   }
-   else
+   // update physical mouse buttons only when they change
+   // this allows the physical mouse and touch mouse to coexist
+   uint64_t mouse_current_report = hidMouseButtonsHeld();
```

Comment:
C89 compliance issue. Variables need to be declared either at the top of a function or at the top of a code block.

Reply:
all fixed in latest force push